app/Http/Controllers/FrontendStatusController.php

```diff
@@ -133,6 +134,8 @@ public function statusesByEvent(string $event): Renderable {
     public function getStatus($statusId): Renderable {
         $status = StatusBackend::getStatus($statusId);
 
+        BrouterController::reroutePolyline($status->trainCheckin->HafasTrip);
```

Comment:
apparently this has to be removed [the person implementing this, sitting next to me is saying that 🤓]